### DISCUSSION TITLE
Chip transfer

### DISCUSSION THREAD

**Discussion by theatomicboy81** [UkrainianBanderasCat/Digital-Logic-Sim2#22] [Ideas]
plz add the ability to transfer a chip from one game to another, this can be useful if you have a complicated chip made in one save and would like it in another but dont want to rebuild it, like an ALU or 4 bit binary to 7 seg or binary to bcd to 7 seg, or massive ram or rom for something tedious like inputing code or letter code.

**Comment by hodytone** [comment]
Yes. This may be useful, but it needs to transfer all chips used recursively. Simple chip(s) transfer can break the project. This may require a merge mechanism.

**Comment by firecerne** [comment]
Made it, it's in the pull request and hopefully will be merged.
